examples/python-guard-shapes/run.sh

```
@@ -0,0 +1,63 @@
+set -uo pipefail
+HERE="$(cd "$(dirname "$0")" && pwd)"
+REPO="$(cd "$HERE/../.." && pwd)"
+BIN="$REPO/implementations/rust/target/debug/provekit"
+
+VENV="${PANDAS_WITNESS_VENV:-/tmp/pandas-witness-venv}"
+if [ ! -x "$VENV/bin/python" ]; then
+  python3 -m venv "$VENV"
+  "$VENV/bin/pip" install -q numpy pandas pytest pynacl blake3 cbor2
+fi
+
+cd "$HERE"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)